module/VuFind/sql/pgsql.sql

```diff
@@ -6,7 +6,7 @@ DROP TABLE IF EXISTS "comments";
 
 CREATE TABLE comments (
 id SERIAL,
-user_id int NOT NULL DEFAULT '0',
+user_id int DEFAULT NULL,
```

Comment:
We should add a pgsql database migration script for these changes, too.

Reply:
Oh, I forgot to add the migration script. It's there now, still untested.